### PR TITLE
Accessibility work

### PR DESCRIPTION
**Pull request by darconny**

### CODE REVIEW

File: assets/home/components/search-bar.jsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ export class SearchBar extends React.PureComponent {
                             <i className="icon--search icon--gray-light" />
                         </span>
                         <div className="search__form input-group searchForm--active">
-                            <form className="form-inline" action="/wire">
+                            <form className="form-inline" action="/wire" role="search" aria-label="search">
```
Comment (petrjasek) [inline assets/home/components/search-bar.jsx:14]: there should be probably gettext used for the label

File: assets/user-profile/components/UserProfileApp.jsx [hunk 2]
```diff
@@ -116,7 +116,7 @@ class UserProfileApp extends React.Component {
                                     {links.find((link) => link.active).label}
                                 </h5>
                                 <div className="content-bar__right">
-                                    <button className="content-bar__menu" aria-label="Close" onClick={this.props.hideModal}>
+                                    <button className="content-bar__menu" aria-label="Close" role="button" onClick={this.props.hideModal}>
```
Comment (petrjasek) [inline assets/user-profile/components/UserProfileApp.jsx:119]: there should be gettext for aria-label

File: newsroom/templates/wire_item.html [hunk 3]
```diff
@@ -8,9 +8,9 @@
     <section class="content-header">
         <div class="content-bar navbar justify-content-between">
 
-            <span class="content-bar__menu">
+            <button class="content-bar__menu" aria-label="Close">
```
Comment (petrjasek) [inline newsroom/templates/wire_item.html:11]: there should be gettext for aria-label

File: assets/home/components/search-bar.jsx (hunk 1 above)
Comment (darconny) [reply assets/home/components/search-bar.jsx:14]: ok let me fix it, tnx

**Review by petrjasek — APPROVED** [review]

**Pull request merged**